rdrf/registry/patients/parent_view.py

```diff
@@ -210,8 +213,12 @@ def post(self, request, registry_code, parent_id, context_id=None):
         else:
             messages.add_message(request, messages.ERROR, "Please correct the errors bellow")
 
+        registry = Registry.objects.get(code=registry_code)
+
         context['parent'] = parent
         context['registry_code'] = registry_code
         context['parent_form'] = parent_form
+        fth = FormTitleHelper(registry, "")
+        context['form_title_dict'] = fth.title_dict_for_user(request.user)
```

Comment:
Can we rename these context variables to `form_titles` and also translate them before putting them in the template.
Then in all templates we could use for example `{{ form_titles.Demographics }}` which reads way better IMO.

Reply:
Updated.